maps/src/test/java/org/odk/collect/maps/layers/MapFragmentReferenceLayerUtilsTest.kt

```diff
@@ -1,17 +1,16 @@
-package org.odk.collect.android.utilities
+package org.odk.collect.maps.layers
 
 import android.os.Bundle
-import androidx.test.ext.junit.runners.AndroidJUnit4
 import org.junit.Assert.assertNotNull
 import org.junit.Assert.assertNull
 import org.junit.Test
 import org.junit.runner.RunWith
-import org.odk.collect.android.geo.DirectoryReferenceLayerRepository
-import org.odk.collect.geo.maps.MapFragment
+import org.odk.collect.maps.MapFragment
 import org.odk.collect.shared.TempFiles.createTempDir
+import org.robolectric.RobolectricTestRunner
 import java.io.File
 
-@RunWith(AndroidJUnit4::class)
+@RunWith(RobolectricTestRunner::class)
```

Comment:
Why did you change this?

Reply:
The `maps` module doesn't currently have any dependencies on AndroidX Test dependencies, so I used `RobolectricTestRunner` directly rather than having `AndroidJUnit4` delegate to it anyway. Annoyingly you can't just include `androidx.test.ext:junit` and use `AndroidJUnit4` in a JVM test without also depending on Robolectric, so because I didn't need anything else out of AndroidX Test I just opted to go with vanilla JUnit and Robolectric.